src/geometry/MergedMeshGroup.ts

```diff
@@ -9,6 +9,31 @@ interface IndexMap { [s: number]: boolean; }
 const globalBox = new THREE.Box3();
 
 export class MergedMeshMappings {
+  public static concat(mappings: MergedMeshMappings[]) {
```

Comment:
Should we abbreviate this or have `concatenate` in full? @dragly @nsunderland-cognite

Reply:
No. https://www.w3schools.com/jsref/jsref_concat_array.asp